stream-chat-android-ui-common/src/main/kotlin/com/getstream/sdk/chat/viewmodel/messages/MessageListViewModel.kt

```diff
@@ -182,7 +189,9 @@ public class MessageListViewModel @JvmOverloads constructor(
                 domain.deleteMessage(event.message).enqueue()
             }
             is Event.FlagMessage -> {
-                client.flagMessage(event.message.id).enqueue()
+                client.flagMessage(event.message.id).enqueue { result ->
```

Comment:
IMO, creating live data aka event bus for async operation here is too much. Event bus can work fine for those events that we don't expect to receive. Here user is source of asyn requests, so if I did implementation I would add to `FlagMessage` new argument `val resultHandler: (Result<Flag>) -> Unit` and wrote this part as:
``` kotlin
 is Event.FlagMessage -> client.flagMessage(event.message.id).enqueue(event::resultHandler)
```

Reply:
You're right, thanks! I was looking for simplest solution and I totally missed that